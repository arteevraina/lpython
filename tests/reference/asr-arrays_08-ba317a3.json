{
    "basename": "asr-arrays_08-ba317a3",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/arrays_08.py",
    "infile_hash": "3f7a5a8889301df17222a777f7e92a5667097ffca9ac4d275ccb310f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-arrays_08-ba317a3.stderr",
    "stderr_hash": "2c0dffd29d0f996f1403e49241e6fe1cfbf7df0ce479e4434c3a7c6a",
    "returncode": 2
}