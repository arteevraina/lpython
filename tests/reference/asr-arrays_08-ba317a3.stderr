warning: The module 'numpy' located in $DIR/src/bin/../runtime/lpython_intrinsic_numpy.py cannot be loaded
 --> tests/errors/arrays_08.py:2:1
  |
2 | from numpy import empty, int64
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ imported here

warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
   --> $DIR/src/bin/../runtime/lpython_intrinsic_numpy.py:364:12
    |
364 |     return x1 % x2
    |            ^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
